### PR TITLE
CC-4408: update java to version 11 for internal services

### PR DESCRIPTION
**Pull request by Deer-WarLord**

### CODE REVIEW

File: Dockerfile.java.in [hunk 1]
```diff
@@ -1,5 +1,6 @@
 FROM zenoss/centos-base:%VERSION%
 MAINTAINER Zenoss <dev@zenoss.com>
 
+RUN yum -y --setopt=tsflags="nodocs" install java-1.8.0-openjdk-headless && /sbin/scrub.sh
```
Comment (jpeacock-zenoss) [inline Dockerfile.java.in:4]: Is there any reason why Java 8 cannot be installed on only the images that require it?

I don't really want to distribute both Java 11 _and_ Java 8 to all images.

**Review by jpeacock-zenoss — CHANGES_REQUESTED** [review]

File: Dockerfile.java.in (hunk 1 above)
Comment (Deer-WarLord) [reply Dockerfile.java.in:4]: Of course, we can do that in each image for the services that require java 11 or java 8. 
But I'd view that as a compromise for a java transition period from 8 to 11.

Comment (jpeacock-zenoss) [reply Dockerfile.java.in:4]: If the serviced-isvcs image started with the non-java centos-base image and installed Java 8, there would be no need to juggle different versions of Java in the configuration.

Comment (Deer-WarLord) [reply Dockerfile.java.in:4]: Yes but we need both version of java for serviced-isvcs image for different containers

**Pull request closed without merging**